includes/view/User_view.php

```diff
@@ -677,11 +677,18 @@ function User_view(
             ]),
             div('row user-info', [
                 div('col-md-2', [
-                    heading(icon('phone')
-                        . '<a href="tel:' . $user_source->contact->dect . '">'
-                        . $user_source->contact->dect, 1)
-                        . '</a>'
-                ]),
+                    heading(
```

Comment:
Eventually a check if `enable_dect` is configured would be nice here

Reply:
done